tests/unit/networked_remote.test.js

```diff
@@ -79,7 +79,6 @@ suite('networked_remote', function() {
 
     test('updates root immediately', sinon.test(function() {
       this.stub(component, 'networkUpdate');
-      this.stub(component, 'waitForTemplateAndUpdateChildren');
```

Comment:
This function no longer exists. It never needed to be stubbed in the first place, as far as I could tell.